docs/release_notes.rst

```diff
@@ -4,6 +4,14 @@ Release Notes
 **pydocstyle** version numbers follow the
 `Semantic Versioning <http://semver.org/>`_ specification.
 
+Current development version
+---------------------------
+
+New features
+
+* Violations are now reported on the line where the docstring starts, not the
+  line of the ``def``/``class`` it corresponds to.
```

Comment:
Link to both this (#238) and #83.

Sorry for the nitpicking, in a few moments this will be merged :)

Reply:
Done.